.github/workflows/AndroidCIWithGmd.yaml

```diff
@@ -0,0 +1,34 @@
+name: Android CI with GMD
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+
+jobs:
+
+  android-ci:
+    runs-on: macos-latest
+
+    steps:
+    - uses: actions/setup-java@v3
+      with:
+        distribution: 'temurin'
```

Comment:
I don't know if it matters, but what is the reason for `temurin` here? I think in our other workflows we have `zulu`

Reply:
I don't have a strong opinion here either. Let's stick with other workflow files. Changed this to `zulu`.